cpp/src/arrow/engine/substrait/serde_test.cc

```diff
@@ -121,6 +155,57 @@ inline compute::Expression UseBoringRefs(const compute::Expression& expr) {
   return compute::Expression{std::move(modified_call)};
 }
 
+// TODO: complete this interface
```

Comment:
Does this still need completed?  Is there a follow-up JIRA?  What's the plan here?

Reply:
Removed this method and replaced with NamedTable. And this could be re-written later when we check file-system based things.